client/app/scripts/liveblog-edit/posts.service.js

```diff
@@ -93,10 +102,24 @@ define([
                 mainItem: post.groups[1].refs[0],
                 items: post.groups[1].refs
             });
-            // complete Post With User Information
-            userList.getUser(post.original_creator).then(function(user) {
-                post.user = user;
+            post.comments = _.reduce(post.groups[1].refs, function(is, val) {
+                return is || _.isUndefined(val.item.name);
+            }, false);
+            // check if `fullDetails` flag is needed
+            // comments items set falg to true.
+            post.fullDetails = post.comments;
```

Comment:
At this logic level, `fullDetails` is not meaning full. Does it means that all the details are filled ? Are they required ? Must they be shown ? And what are details?
Regarding `angular.extend`, it's a matter of preference, but here it has been already used above, so it would be more consistent to continue like that. It also prevent to assign a value twice (jshint will ring the bell)